lib/src/blocs/movies_bloc.dart

```diff
@@ -0,0 +1,26 @@
+import 'dart:async';
+import '../models/item_model.dart';
+import '../resources/movies_repository.dart';
+import 'i_movies_bloc.dart';
+
+class MoviesBloc extends IMoviesBloc {
+  MoviesRepository moviesRepository = MoviesRepository();
+  StreamController<ItemModel> moviesStreamController = StreamController();
```

Comment:
This variables are private inside the Bloc so you need to add at beginning of each an '_'

Reply:
fixed